lib/smart_proxy_pulp_plugin/pulpcore_api.rb

```diff
@@ -26,7 +26,7 @@ class Pulp3Api < Sinatra::Base
         size = :byte
       end
 
-      monitor_dirs = Hash[::PulpProxy::Pulp3Plugin.settings.marshal_dump.select { |key, _| key == :pulp_dir || key == :pulp_content_dir }]
```

Comment:
I was sure we dropped disk usage but looks like we didn't merge https://github.com/theforeman/smart_proxy_pulp/pull/20. Perhaps together they make a nice 2.0?

Reply:
sure, that sounds good